kong/api/init.lua

```diff
@@ -7,125 +7,142 @@ local api_helpers = require "kong.api.api_helpers"
 local tablex = require "pl.tablex"
 
 local find = string.find
-
 local app = lapis.Application()
-
 local needs_body = tablex.readonly({ PUT = 1, POST = 2, PATCH = 3 })
```

Comment:
It's good that we are switching to the new code style in this module, but then let's also respect the 2 lines jump here as well: between the `require` section, the "globals caching section", the "Lapis App creation", and the constants definition for this module.